A visualisation of the quicksort algorithm

SpectralFlame opened the pull request:
Started from the backup code, not too many changes needed to be made

- SpectralFlame aka SpectralPiano

SpectralFlame committed:
A visualisation of the quicksort algorithm

Started from the backup code, not too many changes needed to be made

shiffman commented:
Oh, what a good idea! And this could be done with the Processing native `IntList` or `FloatList` which might make it a bit easier!

SpectralFlame commented:
Happy to see my code might be useful :)
I'll upload a new version later today and I'd love to see your reaction on
that one (perhaps live ^^).
Is the next live stream next friday? I hope I'll catch that!


On Sat, 8 Sep 2018, 03:50 Daniel Shiffman, <notifications@github.com> wrote:

> Oh, what a good idea! And this could be done with the Processing native
> IntList or FloatList which might make it a bit easier!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CodingTrain/Sorting-Viz/pull/5#issuecomment-419604190>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AgHi8RE-jd6wVOpub6_ofJEaZdiDA0fIks5uYyKBgaJpZM4WfimB>
> .
>

SpectralFlame committed:
Hungarian Quicksort Dance

Try sync it with the real video at https://www.youtube.com/watch?v=ywWBy6J5gz8
Enjoy! ^^

SpectralFlame committed:
Added mp3, needs to be converted to wav

Convert the file to WAV (I used https://audio.online-convert.com/convert-to-wav)
make sure the filename is "song.wav" :)

SpectralFlame committed:
Fixed some tiny bugs and sync issues

The syncing is not consistent on my machine which makes it pretty hard to sync it correctly but I think it's quite good now.
